app/src/components/TipCarousel.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { memo, useEffect, useRef, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, View, Text, useWindowDimensions, FlatList, ListRenderItem } from 'react-native'
+
+// used for randomizng tip order
+const shuffleArray = (arr: number[]) => {
+  for (let i = arr.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1))
+    const temp = arr[i]
+    arr[i] = arr[j]
+    arr[j] = temp
+  }
+
+  return arr
+}
+
+// increase / decrease the count in this array if you remove or add tips from the i18n files
+const tipOrder: number[] = shuffleArray([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14])
+
+interface ITip {
```

Comment:
I don't know if we're following the convention of prefixing interfaces with 'I'.

Reply:
Could call this `TipData` or `TipItem` or something like that

Reply:
Good point, changed